src/dispatch/project/models.py

```diff
@@ -40,6 +40,8 @@ class Project(Base):
         cascade="all, delete-orphan",
     )
 
+    display_name = Column(String, nullable=True)
```

Comment:
will this stay nullable or will it be non-nullable post-migration?

Reply:
Good point. I don't think it should be nullable. I fixed this and the database migration.